rosdep/base.yaml

```diff
@@ -3221,6 +3221,9 @@ libqglviewer-qt4:
     xenial: [libqglviewer2-qt4]
     yakkety: [libqglviewer2-qt4]
     zesty: [libqglviewer2-qt4]
+libqglviewer-qt5:
```

Comment:
The CI is complaining because this should be after the qt4 rule.

Reply:
I'll fix that. I tried the `rosdep_format` tool but it only could get it to work for the distribution files, not the rosdep/base.